configs/motion-mmalcam.conf

```diff
@@ -0,0 +1,732 @@
+############################################################
```

Comment:
This config file should probably be processed (generated) as it is done with motion-dist.conf.in during configure